docs/blog/newsletter-november-2020.rst

```diff
@@ -0,0 +1,141 @@
+
+Finally, a few handy WTD resources: `lists of resources for inclusive writing and accessibility tooling <https://www.writethedocs.org/guide/writing/style-guides/#reducing-bias-in-your-writing>`_, plus `some relevant conference talks and writeups in newsletters </topics/#accessibility-and-inclusivity>`__. 
+
+------------------------------------------
+Dealing with stress & struggling to write
+------------------------------------------
+
+Is it any surprise that we've been talking a lot about stress recently? Truly working out how to deal with stress is so individual, and means looking at the big picture - it's not something that you can sort out with one or two quick tips. But our community members recently shared some thoughts on how they manage stress, especially when that stress causes you to struggle to focus and complete work. Maybe some of them you'll find helpful.
+
+* If you're feeling overwhelmed by the weight of your to-do list, perhaps you can explicitly ignore/say no to some work. Can you postpone or drop lower-priority tasks? Or can you ignore them even for a day - just pick one item from your list, and try to complete that? Sometimes finishing one thing can feel like a big win.
+* Collaborate! Your colleagues might be able to help - can you find ways to delegate, or share the load?
+* Recognise when something might be good enough for now. When you have a completed draft, send it out for some feedback, even if it isn't perfect.
+* Take a step back, and intentionally move through each step of the writing process. How would you advise a beginner to work through a task?
+* It's not always easy, but are you looking after yourself? Adequate sleep and nutrition won't fix everything, but they are the solid foundation that we humans need.
+* For feeling overwhelmed in the moment, recognise that you're just not gonna get any work done, and take the break you need. Some people find meditation and/or breathing exercises helpful; other people choose exercise - even if just a few minutes. (My personal favourite? Lie down and take a proper nap - Ed.)
+
+----------------
+From our sponsor
+----------------
+
+
+This month’s newsletter is sponsored by StepGuide:
+
+.. raw:: html
+
+    <hr>
+    <table width="100%" border="0" cellspacing="0" cellpadding="0" style="width:100%; max-width: 600px;">
+      <tbody>
+        <tr>
+          <td width="75%">
+              <p>
+
+              Using <a href="https://www.stepguide.io/">StepGuide</a> is the fastest way to create training docs for all your web applications. What took hours now takes minutes. StepGuide's Chrome Extension records your clicks while using your application and automatically creates step-by-step documentation.
+              </p>
+          </td>
+          <td width="25%">
+            <a href="https://www.stepguide.io">
+              <img style="margin-left: 15px;" alt="StepGuide" src="/_static/img/sponsors/stepguide.png">
```

Comment:
This image is missing, need to be added to the PR

Reply:
I pushed it up

Reply:
Thanks, I had forgotten!